nginx-agent-docker/Dockerfile

```diff
@@ -4,19 +4,17 @@ ARG NMS_URL
 ARG DEVPORTAL=false
 ARG NAP_WAF=false
 
+# Startup script
```

Comment:
You might want to keep the startup script below the initial RUN statement. That way if you make any changes to the script you can still take advantage of the cached layer 😄

Reply:
Good idea, done :)